README.md

```diff
@@ -121,7 +121,7 @@ In this step of our exercise, we will add a "build" job. We will specify `ubuntu
        name: Post welcome comment
        runs-on: ubuntu-latest
    ```
-3. Click **Start commit** in the top right of the workflow editor.
+3. Click **Commit changes** in the top right of the workflow editor.
 4. Type your commit message and commit your changes directly to your branch.
```

Comment:
"Start commit" is actually right. This is the button we find in the top right of the workflow editor. There is no need for changes in item 3.

On the other hand, "Commit changes" is the button we will click on item 4, after typing the commit message. So, item 4 can be re-written for clarity:

```
4. Type your commit message and click **Commit changes** to commit your changes directly to your branch.
```

For quick check:

![image](https://user-images.githubusercontent.com/86871991/202951187-73dfebaf-a4b9-4fb8-80e0-8f98e1d78068.png)